src/chart/sankey/SankeyView.ts

```diff
@@ -195,30 +196,32 @@ class SankeyView extends ChartView {
 
             curve.useStyle(lineStyleModel.getItemStyle());
             // Special color, use source node color or target node color
-            switch (curve.style.fill) {
-                case 'source':
-                    curve.style.fill = edge.node1.getVisual('color');
-                    curve.style.decal = edge.node1.getVisual('style').decal;
-                    break;
-                case 'target':
-                    curve.style.fill = edge.node2.getVisual('color');
-                    curve.style.decal = edge.node2.getVisual('style').decal;
-                    break;
-                case 'gradient':
-                    const sourceColor = edge.node1.getVisual('color');
-                    const targetColor = edge.node2.getVisual('color');
-                    if (isString(sourceColor) && isString(targetColor)) {
-                        curve.style.fill = new graphic.LinearGradient(
-                            0, 0, +(orient === 'horizontal'), +(orient === 'vertical'), [{
-                                color: sourceColor,
-                                offset: 0
-                            }, {
-                                color: targetColor,
-                                offset: 1
-                            }]
-                        );
-                    }
-            }
+            applyCurveStyle(curve.style, orient, edge);
+            // switch (curve.style.fill) {
+            //     case 'source':
+            //         curve.style.fill = edge.node1.getVisual('color');
+            //         curve.style.decal = edge.node1.getVisual('style').decal;
+            //         break;
+            //     case 'target':
+            //         curve.style.fill = edge.node2.getVisual('color');
+            //         curve.style.decal = edge.node2.getVisual('style').decal;
+            //         break;
+            //     case 'gradient':
+            //         const sourceColor = edge.node1.getVisual('color');
+            //         const targetColor = edge.node2.getVisual('color');
+            //         if (isString(sourceColor) && isString(targetColor)) {
+            //             curve.style.fill = new graphic.LinearGradient(
+            //                 0, 0, +(orient === 'horizontal'), +(orient === 'vertical'), [{
+            //                     color: sourceColor,
+            //                     offset: 0
+            //                 }, {
+            //                     color: targetColor,
+            //                     offset: 1
+            //                 }]
+            //             );
+            //         }
+            // }
+
```

Comment:
Just please clean the commented code fragement.

Reply:
removed commented code